modules/table/src/main/java/org/apache/ignite/internal/table/distributed/storage/RowBatch.java

```diff
@@ -31,19 +31,19 @@
  *
  * <p>NOTE: Not thread-safe.
  */
-class RowBatch {
+public class RowBatch {
```

Comment:
Why public?

Reply:
Because we have a refernce on UpdatableTableImpl.

Reply:
Then it needs to add encapsulation.

Reply:
Comment ignored.